demo/b2share_demo/data/communities/euon.json

```diff
@@ -81,7 +85,8 @@
                                     "OntoStudio",
                                     "KAON",
                                     "PoolParty Thesaurus Server",
-                                    "Other..."
+                                    "Other...",
+									""
```

Comment:
This is required to pass validation on usedOntologyEngineeringTool="" which appears and to my opinion we cannot map the case "No tool used" to "Other tool used" ? So I keep the value. Note that usedOntologyEngineeringTool is not a required field in this schema, so we can assume that "empty" indeed means "No tool used" (as would leaving out the key/value all together imply too).